### DISCUSSION TITLE
Invoke a script from the installer

### DISCUSSION THREAD

**Discussion by perevodchyk** [wixtoolset/issues#7533] [Questions]
I am trying to implement a script which removes temporary files generated by the installer. (https://github.com/orgs/wixtoolset/discussions/7312)

The script needs to be run asynchronously because tmp files are in use until I exit the wizard.

I would like to launch a script from the installer in the following way:

`<CustomAction Id="DeleteTempFiles" ExeCommand="wscript.exe delete_temp_files.vbs" BinaryKey="B.deleteTempFilesScript" Execute="deferred" Return="asyncNoWait" />`

However, nothing happens, as if the script didn't exist.
What path should I provide in 'ExeCommand'?

**Comment by robmen** [comment]
That sounds like a really bad idea. It is not your responsibility to clean up those files, although it does sound like you should refactor your code to minimize the impact of the files. Fundamentally, that isn't your folder to go deleting stuff.
